### DISCUSSION TITLE
Difference between /pages/api/something.js and /pages/something.js (with getServerSideProps) in Next.js on Vercel

### DISCUSSION THREAD

**Discussion by mzalevski** [vercel/next.js#15122] [Help]
[Next.js on Vercel] Hello, is there any difference between SSR page and a function placed in /pages/api? thinking both can return json data and both are turned into serverless functions.

I suppose that on Vercel there is no difference afterall, but on node.js server target outside of Vercel there is no /pages/api, so only SSR pages are left (and they are ofc running not in a serverless manner). Is this correct?

"Pages that use Server-Side Rendering and API routes will automatically become isolated Serverless Functions." https://nextjs.org/docs/deployment

**Comment by timneutkens** [comment]
API routes work fine when running `next start` as well, they're **not** limited to Vercel.

**Reply by mzalevski** [reply]
Got it. So it seems that, in terms of functionality, there is no difference between SSR page and a function placed in /pages/api. Only the syntax differs. Right?
